packages/ui/src/lib/components/WalletButton.svelte

```diff
@@ -11,6 +11,7 @@
   const dispatch = createEventDispatcher();
   export let extensionMetadata: ConfiguredExtension;
 
+  /* eslint svelte/valid-compile: "off" */
   $: cachedExtension = $CachedExtensionsStore?.[extensionMetadata.injectedName] ?? {
```

Comment:
This is read only reactive, so it is fine, but svelte doesn't like it. If it wasn't an old version I might update, but this is not the time to do that.